modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-buildah-build.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-buildah-build-in-a-network-restricted-environment_{context}']
+= Creating buildah build in a network restricted environment
+
+You can create a `buildah` build in a network network restricted environment by mirroring the images required by the `buildah` buildStrategy and then creating a `buildah` build. 
+
+.Prerequisites
+
+* Your cluster can connect and interact with the git source that you will use to create the buildah build.
+
+.Procedure
+
+. Run the following command to mirror the images required by the buildah buildStrategy:
++
+[source,terminal]
+----
+$ oc image mirror --insecure -a <registry-authentication> registry.redhat.io/ubi8/buildah@sha256:1c89cc3cab0ac0fc7387c1fe5e63443468219aab6fd531c8dad6d22fd999819e <mirror-registry>/<repo>/ubi8_buildah
```

Comment:
I am not sure `--insecure` should be used here. I think it was only for testing. Let me get back regarding this.

Reply:
Okay, Sayan confirmed that it can be used.